docs/upgrading.md

```diff
@@ -8,12 +8,77 @@ the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summar
 
 ### feat!: Remove deprecated config flags
 
-This PR removes the following options -
+This PR removes the following configmap items -
 
 - executorImage (use --executor-image flag to workflow-controller instead)
+  e.g.
+  Workflow controller configmap like the one given below won't be valid because of using executorImage key -
+
+  ```yaml
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: workflow-controller-configmap
+  data:
+    instanceID: my-ci-controller
+    ...
+    executorImage: argoproj/argocli:latest
+    ...
+  ```
+
+  From now and onwards, only provide the executor image in workflow controller as a command argument as shown below -
+
+  ```yaml
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: workflow-controller
+  spec:
+    ...
+    template:
+      ...
+      spec:
+        containers:
+        - args:
+          - --configmap
+          - workflow-controller-configmap # Your configmap name
```

Comment:
I’m not sure I’d recommend using cli parameters. It works very poorly with Kustomize patches. I don’t use them myself. Can you instead show the change to make to the configmap?

Reply:
Oh so in that case I will need to use the image key of executor? Should I change this line as well?

> use --executor-image flag to workflow-controller instead

I found it in deprecation notes so was using it.

Reply:
yes - both